3515/2/commands/checkout.rb

```diff
@@ -0,0 +1,85 @@
+require_relative '../helper_modules/directory_maker'
+require_relative '../helper_modules/file_write'
+module CheckoutCommand
+  def checkout!(*)
+    if user_authorized?
+      checkin_true?
+    else
+      respond_with :message, text: "Извините, но ваш аккаунт не авторизован\nДля авторизации
+      введите команду /start"
+    end
+  end
+
+  def user_authorized?
```

Comment:
Hmmm, seems like I've already seen this method before... There are a few ways to share the common logic: parent class or mixins for example. It's up to you to choose the way, but I'd like to not have this copy-paste